[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\Owner\\AppData\\Local\\Temp\\quarto-sessionf488153661d0a3c8'\n"]
